archiving-of-web-and-social-media.py

```diff
@@ -473,4 +427,98 @@ def main():
 
 
 if __name__ == "__main__":
-    main()
+    # Config section ###################################################################
+
+    # Create a .env file and put your social media usernamnes and password there.
+    # load_dotenv()
+    # instagram_user = os.getenv("instagram_user")
+    # instagram_password = os.getenv("instagram_password")
+    # linkedin_user = os.getenv("linkedin_user")
+    # linkedin_password = os.getenv("linkedin_password")
+    # facebook_user = os.getenv("facebook_user")
+    # facebook_password = os.getenv("facebook_password")
+
+    # type_of_web_extraction = "gislaved.se"
+    # gislaved.se
+    # insidan.gislaved.se
+    # Facebook
+    # LinkedIn
+    # Instagram
+
+    width_of_screenshot = 1920
+    headless_for_full_height = True  # Adjust this to true to get full height. False för debugging to see how buttons are clicked
+    xsd_file = "FREDA-GS-Webbsidor-v1_0.xsd"  # XSD file for validation of FGS. Change to your own XSD.
+    contract = "Contract_2020-02-24-13-03-23-WEB.xml"  # contract file for LTA upload
+    systemnamn = "Webbsidor"  # If you want package creator systemnamn to be the basmetadata "Ursprung" instead set this to empty string ("")
+
+    # Load the Excel file with a list of web pages for the current run
+    pages_to_crawl_file = 'pages_gislaved_se_extern_webb.xlsx'
+    # The following is the the two columns of the pages excel.
+    # Webbadress	Webbsida
+    # The first is the url to be crawled
+    # The second is a short description of the url that goes in the FGS node webbsida
+
+    basmetadata_file = 'basmetadata_extern_webb.xlsx'
+    # The following is the first column of the excel with basmetadata. The second column is for the values
+    # Basmetadata
+    # Organisation
+    # Arkivbildare
+    # Arkivbildarenhet
+    # Arkiv
+    # Serie
+    # Klassificeringsstruktur
+    # nivå1
+    # nivå2
+    # nivå3
+    # Ursprung
+    # Sekretess
+    # Personuppgifter
+    # Forskningsdata
+    # Kommentar
+
+    # End config section #############################################################
+
+    print("Welcome to Mediahanteraren")
+
+    while True:
+        print("************************************")
+        print("Type 'Exit' to quit at any time.")

```

Comment:
Det kanske finns automatiskt redan, men om inte så gör att man kan trycka ett kortkommando för att avsluta också, ctrl-c kanske? Och skriv ut det med som alternativ.

Reply:
Från ChatGPT: Combining Ctrl+Q with Python's input() function to exit without pressing Enter is challenging because input() inherently waits for the user to hit Enter to terminate input.